### PR TITLE
deb rpm: verify to install the variants of apt-source and .repo

### PR DESCRIPTION
**Pull request by kenhys**
* deb rpm: verify to install the variants of apt-source
* rpm: follow repository layout for redhat variants

### CODE REVIEW

File: fluent-package/apt/systemd-test/test.sh [hunk 1]
```diff
@@ -19,11 +19,18 @@ fi
 set -eu
 
 test_filenames=(
-    install-newly.sh
     update-from-v4.sh
     update-to-next-version.sh
 )
 
+for apt_repo_type in local v5 lts; do
+    echo -e "\nRun test: $repo_type\n"
```
Comment (daipom) [inline fluent-package/apt/systemd-test/test.sh:27]:
```suggestion
    echo -e "\nRun test: $apt_repo_type\n"
```

File: fluent-package/yum/systemd-test/test.sh [hunk 2]
```diff
@@ -19,11 +19,17 @@ fi
 set -eu
 
 test_filenames=(
-    install-newly.sh
     update-from-v4.sh
     update-to-next-version.sh
 )
 
+for yum_repo_type in local v5 lts do
+    echo -e "\nRun test: $test_filename\n"
```
Comment (daipom) [inline fluent-package/yum/systemd-test/test.sh:27]:
```suggestion
    echo -e "\nRun test: $yum_repo_type\n"
```

Comment (kenhys) [reply fluent-package/yum/systemd-test/test.sh:27]: Thanks

**Comment by kenhys** [comment]
Fixed bogus typo.

**Review by daipom — APPROVED** [review]
Thanks! Looks good to me.

I'm running yum tests without the serverspec test on my fork: https://github.com/daipom/fluent-package-builder/actions/runs/5688587254

**Pull request merged**